### ISSUE TITLE
files in /etc/ld.so.conf.d/ contain wrong paths

### ISSUE THREAD

**Issue by clavinet** [RadeonOpenCompute/ROCm#1156]
System: openSUSE Tumbleweed, but should apply to SUSE too

During installation, ROCm packages will create files in /etc/ld.so.conf.d/, for example:
/etc/ld.so.conf.d/hsa-rocr-dev.conf

This file points to a nonexistent location:
```
$ cat hsa-rocr-dev.conf 
/opt/rocm/hsa/lib
```

/opt/rocm doesn't exist anymore, since all rocm stuff is in versioned folders like /opt/rocm-3.5.1 now.

**Comment by ableeker** [comment]
The installation guide tells us we can install the entire package, or a number of components, in a number of ways. If you install rocm-dkms, or rocm-dev, you're installing the complete package, and it will create the symlink /opt/rocm by itself. If you install some components, you can for instance just install rocm-opencl, it won't create /opt/rocm. In this case you'll have to create it yourself. You can install different versions of ROCM as well. If you do this, you'll also have to create /opt/rocm, and point it to the version you want to use.

In short, /opt/rocm should exist, or you'll have to create it yourself.

**Comment by clavinet** [comment]
Thanks, I installed rocm-dev and it created the symlink.
